api/client/samples/crop_models/sao_paulo_sugar_content/Sao_Paulo_brazil_sugar_content_modeling.ipynb

```diff
@@ -0,0 +1,3628 @@
+{
```

Comment:
<p>replace crop_id with item_id since in this case the item is not the crop but the trs of the crop</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Done</p>